### PR TITLE
2015-11-11-ruby-2-3-0-preview1-released (ru)

### PR DESCRIPTION
**Pull request by razum2um**
/cc @gazay @arbox

### CODE REVIEW

**Comment by gazay** [comment]
:+1: @arbox any comments?

File: ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+title: "Вышел Ruby 2.3.0-preview1"
+author: "naruse"
+translator: "Vlad Bokov"
+date: 2015-11-11 14:00:00 +0000
+lang: ru
+---
+
+Мы рады сообщить о выходе Ruby 2.3.0-preview1.
+
+Ruby 2.3.0-preview1 - первая предварительная версия релиза Ruby 2.3.0,
```
Comment (arbox) [inline ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md]: Probably `&mdash;` ?

File: ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+* <https://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.0-preview1.zip>
+
+  * SIZE:   19777010 bytes
+  * SHA1:   3458666c250dd333ac4b537be25f1913dd311ea1
+  * SHA256: 0fc1d4450e9141f541038a4433f8fb49fb929e43c7bf5985685a4fa7b213a811
+  * SHA512: f53653cf11d8dce1d627f7db3a15acca29c74c78735406a5d64bbbdc39fac232ef63110eb4d83521bef40c6f3006d55bbeaf54d1ddab1d4b6e5a13cee9247e85
+
+## Комментарий к релизу
+
+Также, вы можете посмотреть план выхода новых версий и получить другую информацию по
```
Comment (arbox) [inline ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md]: No comma.

File: ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+[ссылке](https://bugs.ruby-lang.org/projects/ruby-trunk/wiki/ReleaseEngineering23),
```
Comment (arbox) [inline ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md]: Dot at the end.

File: ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+Гем [did_you_mean вошел в стандартную поставку](https://bugs.ruby-lang.org/issues/11252).
+Эта библиотека показывает подходящие варианты исправлений после ошибок `NameError` и `NoMethodError`
+для облегчения отладки.
+
+Попробуйте и ощутите радость от программирования с Ruby 2.3.0-preview1,
+а также поделитесь с нами своим опытом!
+
+## Значительные изменения с 2.2
+
+См. подробнее [новости](https://github.com/ruby/ruby/blob/v2_3_0_preview1/NEWS) и
```
Comment (arbox) [inline ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md]: Do we really need an abbreviation?

File: ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+lang: ru
+---
+
+Мы рады сообщить о выходе Ruby 2.3.0-preview1.
+
+Ruby 2.3.0-preview1 - первая предварительная версия релиза Ruby 2.3.0,
+которая включает много новых возможностей и улучшений.
+
+Введен концепт [иммутабельности строковых литералов](https://bugs.ruby-lang.org/issues/11473).
+
```
Comment (arbox) [inline ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md]: Иммутабельный? I would say "неизменяемый", which is also reflected in Wikipedia: https://ru.wikipedia.org/wiki/%D0%9D%D0%B5%D0%B8%D0%B7%D0%BC%D0%B5%D0%BD%D1%8F%D0%B5%D0%BC%D1%8B%D0%B9_%D0%BE%D0%B1%D1%8A%D0%B5%D0%BA%D1%82

File: ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+В Ruby 2.1 метод `"str".freeze` бы оптимизирован, чтобы уменьшить количество выделений памяти.
+В Ruby 2.3 появляется магический комментарий и опция коммандной строки, чтобы "заморозить"
```
Comment (arbox) [inline ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md]: был

**Comment by arbox** [comment]
Nice work! :+1:

File: ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md (hunk 6 above)
Comment (razum2um) [inline ru/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md]: hm?
> Ruby 2.3 introduces a new magic comment and command line option

**Comment by razum2um** [comment]
@arbox thanks for checking :)

**Comment by gazay** [comment]
Thanks @razum2um @arbox !

**Pull request merged**